src/main/java/org/jenkinsci/plugins/gpjenkins/GlobalizationPipelineBuilder.java

```diff
@@ -803,23 +821,11 @@ else if(!workspace.child(outDir.trim()).isDirectory()){
 					}
 					
 					
-					ClassLoader orig = Thread.currentThread().getContextClassLoader(); 
-					Thread.currentThread().setContextClassLoader(Jenkins.getInstance().getPluginManager().uberClassLoader); 
-					ServiceLoader.load(CSVFilter.class); 
-					/* 
-					 * (optional - only needed in case of adding your own custom filters)
-					 * ADD more custom filters as needed with your custom filter provider class
-					 * Above example uses CSV filter for example.
-					 * For e.g ServiceLoader.load(MyCustomFilter.class);
-					 */
-					
-					
 					// Parse the resource bundle file
 					ResourceFilter filter = ResourceFilterFactory.getResourceFilter(getType());
 					Thread.currentThread().setContextClassLoader(orig); 
```

Comment:
You might move the above line outside the condition too

Reply:
moved